### ISSUE TITLE
search form: categorising institutions

### ISSUE THREAD

**Issue by Lilith-Palmer** [CodeforAustralia/school-finder#10]
Categorising entrants to match with available institutions- specifically, we have three types of institutions, but two of them overlap (k-2 and k-6).

These need to be remedied with the options available to the end-user on the initial search. Currently we ask for 'school type' with two options available - primary or secondary. Results include K-2 and k-6 institutions for primary, and 7-12 instutions for secondary.

*alter question to ask for grade rather then school type 
*increase options to include three choices-k-2, 3-6, 7-12
*edit reults to show infants and primary instutions for the k-2 selection, primary only for the 3-6 selection and secondary only for the 7-12 selection

**Comment by techieshark** [comment]
@Lilith-Palmer it's actually even more complicated; more than two are overlapping:

![image](https://cloud.githubusercontent.com/assets/1072292/6723351/d61707a8-ce3a-11e4-9367-a588ab57d3ea.png)

(You can ignore BD, EEC, ED, M/HSN)

**Comment by Lilith-Palmer** [comment]
@techieshark A lot more then two! Where are these eight categories coming from? Is it the format the data is provided in? Would be much better to consolidate these from the source (if possible). Otherwise we may need to reorganise our selector.

**Comment by techieshark** [comment]
@Lilith-Palmer yeah that's the raw data (and below, with counts of how often we see that). Seems like the only thing that'd work is letting the user choose one grade (out of all the options, K-12).

count | level_of_schooling | subtype
--------|---------------------------|------------
1592 | Primary School | K to 6
363 |Secondary School |7 to 12
66 | Central/Community School | K to 12
21 | Secondary School | 7 to 10
17 | Infants School | K to 2
12 | Secondary School | 11 to 12
5 | Schools for Specific Purposes | M/HSN
4 | Secondary School | 10 to 12
4 | Environmental Education Centre | EEC
1 | Schools for Specific Purposes | ED
1 | Secondary School | 7 to 9
1 | Schools for Specific Purposes | BD

**Comment by Lilith-Palmer** [comment]
@techieshark I agree- single selection with a search that return all options based on that grade. Drop down option or buttons?

**Comment by techieshark** [comment]
@Lilith-Palmer and actually… we've got preschools too:

![image](https://cloud.githubusercontent.com/assets/1072292/6724802/966fd4aa-ce52-11e4-8d95-a49799b5b2b0.png)

````
SELECT count(*) FROM dec_schools WHERE preschool_indicator = true
````
shows that there are 99 schools with preschools.

For specifics: 

count | level_of_schooling | subtype
--------|---------------------------|-----------
91 | Primary School | K to 6
4 | Infants School | K to 2
4 | Central/Community School | K to 12

**Comment by Lilith-Palmer** [comment]
@techieshark these numbers are for preschools attached to another institution only- therefore should form part of individual institution breakdown, rather than an inital criteria? If included in the grade selection, could be misleading as it isn't showing all preschools in an area; only those attached to a school.
